contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -100,6 +102,12 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, ERC677Recei
   // ================================================================
   // |                      Getter methods                          |
   // ================================================================
+  /**
+   * @inheritdoc IFunctionsSubscriptions
+   */
+  function getMaxConsumers() external pure override returns (uint16) {
```

Comment:
Is this even needed?

Reply:
Maybe not. It could be internal/private. If I have to reduce contract sizes again it will go away.